.tekton/thanos-1-1-pull-request.yaml

```diff
@@ -197,8 +197,6 @@ spec:
             value: $(params.output-image).prefetch
           - name: ociArtifactExpiresAfter
             value: $(params.image-expires-after)
-          - name: dev-package-managers
-            value: $(params.prefetch-dev-package-managers-enabled)
```

Comment:
Hmm the task params shouldn't need changing. what am I missing? Afaict this is just how the pipeline tasks works.
Otoh this doesn't seem to hurt for now so feel free to go ahead and merge.

Reply:
Hi, as we'll no longer use the rpm dev package manager we won't be needing for now to enable this at all so I thought about reducing the noise and just keep it off ;) 

This needed a rebase in any case so I'll merge once CI is done, thanks!